package.json

```diff
@@ -30,7 +30,14 @@
   },
   "homepage": "http://material-ui.com/",
   "dependencies": {
-    "react-draggable2": "^0.5.1"
+    "react": ">=0.14.0-rc1",
+    "react-dom": ">=0.14.0-rc1",
```

Comment:
I guess with the new npm something has to be done for test, we cannot add peer dependencies here as dependencies.

Either:
- we find some way to install peer dependencies for test using npm
- we create a test project with a package.json that brings them as dependencies?